src/components/manage_users_table/ManageUsersTableCellData.js

```diff
@@ -17,51 +34,51 @@ export function usernameCellData({displayName, userId, label= 'user-name'}) {
   };
 }
 
-export function googleIdCellData({userId, email, label = 'google-id'}) {
+export function emailCellData({userId, email, label = 'email'}) {
   return {
     data: email,
     value: email,
     id: userId,
     style: {
-      color: styles.color.googleId,
-      fontSize: styles.fontSize.googleId,
+      color: styles.color.email,
+      fontSize: styles.fontSize.email,
       fontWeight: '500'
     },
     label
   };
 }
 
-//Redirect for admin review page, only used in admin manage dar collections table
-export function roleCellData({userId, roles, label = 'user-role'}) {
+export function permissionsCellData({userId, roles, libraryCards, label = 'permissions'}) {
+  const hasLibraryCard = !isNil(libraryCards) && !isEmpty(libraryCards);
+  const roleNames = map(roles, 'name').filter((roleName) => roleName !== 'Researcher');
+  const perms = (hasLibraryCard ? roleNames.concat('LibraryCard') : roleNames);
+
+  // need to split, e.g., SigningOfficial -> Signing Official
+  // use regex to split by case, then stitch back together with spaces
+  // regex from: https://gist.github.com/JeffJacobson/3841577
+  const splitByWordRegex = /($[a-z])|[A-Z][^A-Z]+/g;
+  const formattedPerms = perms.map((perm) => perm.match(splitByWordRegex).join(' '));
```

Comment:
Nice!

Reply:
I was digging more into this and found a [nice library](https://vocajs.com/#) that does this work as well:

```
const v = require('voca');
const val = v.words('DataSubmitter').join(' ');
console.log(val);
```
Outputs: `Data Submitter`

Reply:
assuming this is the preference, i added voca and used it to split here!

Reply:
I don't have a preference, either way works!